test/integration/programs/sidebar/action-sidebar.js

```diff
@@ -794,7 +794,7 @@ context('program action sidebar', function() {
       .intercept({
         method: 'PATCH',
         url: 'api/program-actions/1',
-      }, { statusCode: 201 })
+      }, { statusCode: 204 })
```

Comment:
These are 204s when no content on the backend.  201s are for successful POSTs and return data